signup.css

```diff
@@ -0,0 +1,131 @@
+.signup{
+    display: grid;
+    grid-template-columns: 50% 50%;
+    background-color: var(--light-blue);
+    border-radius: 8px;
+    width: 90%;
+    max-width: 93.7rem;
+    margin: 0rem auto;
+    padding: 5rem 1rem;
+    box-shadow: rgba(0, 0, 0, 0.24) 0px 3px 8px;
+    gap: 1rem;
+}
+.wrapper{
+    position: relative;
+}
+.blob{
+    position: absolute;
+    width: 30%;
+    z-index: -1;
+}
+.blob1{
+    top: -10px;
+    left: -100px;
+}
+.blob2{
+    bottom: 100px;
+    right: -100px;
+}
+.authentication-image-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 1 / 2;
+    z-index: 2;
+    width: 100%; 
+}
+.authentication-image-container img{
+    width: 100%;
+    max-width: 40rem;
+}
+.authentication-form-container{
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    grid-column: 2 / 3;
+    z-index: 2;
+}
+.authentication-form{
+    background-color: white;
+    max-width: 31.2rem;
+    width: 100%;
+    border-radius: 6px;
+    padding: 1rem 0.5rem;
+    box-shadow: rgba(0, 0, 0, 0.16) 0px 3px 6px, rgba(0, 0, 0, 0.23) 0px 3px 6px;
+}
+.authentication-form-heading{
+    text-align: center;
+    margin: 0rem 1rem 0.3rem 1rem;
+}
+.authentication-form-sub-heading{
+    margin: 0.3rem 1rem;
+    text-align: center;
+}
+.form-input-box{
+    position: relative;
+    display: flex;
+    justify-content: center;
+    margin: 1.8rem 0rem;
+}
+.form-input-box input{
+    width: 90%;
+    padding-left: 1rem;
+    height: 3.12rem;
+    box-shadow: rgba(0, 0, 0, 0.1) 0px 4px 12px;
+    border-radius: 3px;
+    border: 1px solid #BCBCBC;
```

Comment:
use variables for color

Reply:
Its just a border that is why i used this only